.genx/package.json

```diff
@@ -6,6 +6,7 @@
   "license": "UNLICENSED",
   "devDependencies": {
     "@genesislcap/foundation-cli": "^1.0.0",
-    "yargs-parser": "^20.2.7"
+    "yargs-parser": "^20.2.7",
+    "@types/fs-extra": "^9.0.12"
```

Comment:
is 9.0.12 the latest version?

Reply:
Interesting - that would mean to include the latest 9.x version (minor and patch updates), however, I see in my `node_modules` that the downloaded `fs-extra` is 10.1.0. @skawian any idea what's going on here? Or may be I interpret this wrong?

Reply:
ah, I think I know - this is the typescript bindings I assume? Which brings a specific `fs-extra` version?

Reply:
the above line is for typescript types not the actual library. It's correctly in devDependencies. You should have another one in "dependecies". It could have ended up in your node_modules because some other library depends on it

Reply:
I think it's ok

Reply:
if you're using the library here I would suggest adding types as devDependency here, then adding actual library here as well on the same version. This way you avoid problem of the library being pulled out of foundation-cli and breaking here while doing update